packages/grafana-ui/src/components/Table/TableNG/utils.ts

```diff
@@ -25,105 +26,42 @@ import { getTextColorForAlphaBackground } from '../../../utils/colors';
 import { TableCellOptions } from '../types';
 
 import { COLUMN, TABLE } from './constants';
-import { CellColors, TableRow, ColumnTypes, FrameToRowsConverter, Comparator } from './types';
-
-/* ---------------------------- Cell calculations --------------------------- */
-export type CellNumLinesCalculator = (text: string, cellWidth: number) => number;
+import {
+  CellColors,
+  TableRow,
+  ColumnTypes,
+  FrameToRowsConverter,
+  Comparator,
+  TypographyCtx,
+  LineCounter,
+} from './types';
+
+/* ----------------------------- Cell options and attributes ---------------------------- */
```

Comment:
the order of the utils had gotten super out-of-whack, so I have reordered the file since there's not so many in-flight Table PRs right now that aren't directly related to this one anyway. I think the main one that I think we ought to get in before this would be [the Nanos one](https://github.com/grafana/grafana/pull/108614).